index.bs

```diff
@@ -305,10 +305,9 @@ spec: webidl
         </li>
         <li>
           If the user grants permission, return {{"granted"}}; otherwise return
-          {{"denied"}}. If the user's interaction indicates they intend this
-          choice to apply to other realms, then treat this as <a>new information
-          about the user's intent</a> for other <a>realms</a> with the <a>same
-          origin</a>.
+          {{"denied"}}. The user's interaction may provide <a>new information
+          about the user's intent</a> for this <a>realm</a> and other
+          <a>realms</a> with the <a>same origin</a>.
```

Comment:
Is the implication that this prohibits this from providing new information for other origins?  Because it might be read that way.  I would have thought that saying "NO WAY!" to a prompt on `https://google.com` might reasonably be interpreted as a rejection for (for example) `http://google.com`, and even other ports on the same host.  That's something that I believe we do for some permissions in Firefox.